geyser-plugin-interface/src/geyser_plugin_interface.rs

```diff
@@ -328,6 +328,9 @@ pub enum SlotStatus {
 
     /// A new bank fork is created with the slot
     CreatedBank,
+
+    /// A slot is marked dead
+    Dead,
```

Comment:
Should we include the reason for marking the slot dead as a string in this enum?